backend-api/generate_env_file.sh

```diff
@@ -11,8 +11,8 @@ fi
 
 echo "Generating .env file for the $SAM_STACK stack"
 
-PUBLIC_API_URL=$(aws cloudformation describe-stacks --stack-name "$SAM_STACK" --query "Stacks[0].Outputs[?OutputKey=='PublicApiUrl'].OutputValue" --output text)
+SESSIONS_API_URL=$(aws cloudformation describe-stacks --stack-name "$SAM_STACK" --query "Stacks[0].Outputs[?OutputKey=='PublicApiUrl'].OutputValue" --output text)
```

Comment:
Does this `PublicApiUrl` need to be `SessionsApiUrl`?

Reply:
nice spot - I missed this file in testing